client/lib/ScriptInstance.ts

```diff
@@ -18,7 +18,9 @@ export default class ScriptInstance {
 
   constructor() {
     this.mode = process.env.NODE_ENV as any;
-    if (!['development', 'production', 'multiverse', 'timetravel'].includes(this.mode)) {
+    if (
+      !['development', 'production', 'multiverse', 'timetravel', 'browserless'].includes(this.mode)
```

Comment:
Should this be `headless` instead of `browserless`?

Reply:
I too was tempted by this name... haha. But it's not a headless browser, which is what that sounds like you're getting. It's NO browser context or tab. I had called it background originally too. I imagine this might someday be useful for running network-only requests

Reply:
So it's a basic http fetch? Or more? What is the use-case?

Reply:
It's used currently when we re-run extraction. I wanted a generic way to tell Hero that no Tab or BrowserContext needed to be created, and i needed a way in ChromeAlive Core to detect a session that is NOT going to be run anything with Hero. We could have called this "extract only" mode too, but we have no notion of extraction in Hero, so that felt like it crossed-over too much. I thought the notion of a session with no browser might have future utility.